Formula/y/yass-cli.rb

```diff
@@ -0,0 +1,70 @@
+class YassCli < Formula
+  desc "Lightweight and efficient, socks/http forward proxy"
+  homepage "https://letshack.info"
+  url "https://github.com/Chilledheart/yass/releases/download/1.13.1/yass-1.13.1.tar.bz2"
+  sha256 "26015d32a6bdc5bf2670b36c8f5fb68f4a324d577e027bc5eef18ae977e00c9f"
+  license "GPL-2.0-only"
+  head "https://github.com/Chilledheart/yass.git", branch: "develop"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "llvm" => :build if DevelopmentTools.clang_build_version < 1500
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "c-ares"
+  depends_on "libnghttp2"
+  depends_on "mbedtls"
+
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "zlib"
+  end
+
+  def install
+    ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version < 1500)
+
+    args = %w[
+      -DBUILD_SHARED_LIBS=off
+      -DUSE_BUILTIN_CA_BUNDLE_CRT=off
+      -DCLI=on
+      -DSERVER=off
+      -DGUI=off
+      -DBUILD_TESTS=off
+      -DUSE_SYSTEM_MBEDTLS=on
+      -DUSE_ZLIB=on
+      -DUSE_SYSTEM_ZLIB=on
+      -DUSE_CARES=on
+      -DUSE_SYSTEM_CARES=on
+      -DUSE_SYSTEM_NGHTTP2=on
+    ]
+
+    args << "-DUSE_LIBCXX=on" if OS.mac?
+
+    system "cmake", "-S", ".", "-B", "build", "-G", "Ninja", *(std_cmake_args + args)
+    system "cmake", "--build", "build"
+
+    bin.install "build/yass_cli"
```

Comment:
Why aren't we using the install target?

Reply:
Install target is used for GNU-compatible system such as install systemd files and etc. it will introduce some noise on homebrew directory.

Reply:
That's fine. Let's just it.